starwars/starwars_people/views.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from starwars_people.models import Dataset
+from starwars_people.serializers import DatasetSerializer
+from starwars_people.services import CSVDataWriterAndDBSaver
+
+
+class DatasetDetailView(ModelViewSet):
+    queryset = Dataset.objects.all()
+    serializer_class = DatasetSerializer
+
+    def create(self, request, *args, **kwargs):
+        data = CSVDataWriterAndDBSaver()
+        data.write_data_and_save_to_file()
+        data.save_file_to_db_and_remove_from_disc()
```

Comment:
you need other service here. And for the future, make `write_data_and_save_to_file` and `save_file_to_db_and_remove_from_disc` private, then make one public methid where you call this private methods. (Incapsulation)